daml-foundations/daml-ghc/base-rst-prefix.rst

```diff
@@ -3,8 +3,25 @@
 
 .. _stdlib-reference-base:
 
-Base for DAML 1.2
-=================
+The standard library
+====================
 
-The base domain of the DAML standard library contains various modules with
+The DAML standard library is a collection of DAML modules organised in domains
+that can be used to implement concrete applications.
+
+This page covers the Base domain, which contains various modules with
```

Comment:
It isn’t quite clear to me whether “Base domain” is intended to be a technical term in the sense that “Module” and “Package” are and what exactly it encompasses.

Reply:
I think I blindly copied this from somewhere, and it doesn't make sense - I'll remove it.